runtime/codert_vm/cnathelp.cpp

```diff
@@ -1516,7 +1518,8 @@ old_slow_jitResolveInterfaceMethod(J9VMThread *currentThread)
 		goto retry;
 	}
 	indexAndLiteralsEA[2] = (UDATA)interfaceClass;
-	indexAndLiteralsEA[3] = methodIndexAndArgCount >> 8; /* remove argCount */
+	// store iTable offset instead of index
+	indexAndLiteralsEA[3] = ((methodIndexAndArgCount >> 8) * sizeof(UDATA)) + sizeof(J9ITable);
```

Comment:
Does this need to clear the tag bits: `J9_ITABLE_INDEX_METHOD_INDEX | J9_ITABLE_INDEX_OBJECT` ?

Reply:
This code writes the offset into `indexAndLiteralsEA[3]` but the other code deals with `[1]`.  Am I missing something?

Reply:
One of the helpers passes the head of the data, the other passes the middle of the data (for historical reasons).

The bits will be dealt with in the follow-up PR - they are currently never set.